static/js/index.js

```diff
@@ -1050,6 +1050,18 @@ EpComments.prototype.saveComment = async function (data, rep) {
   const [commentId, comment] = res;
   comment.commentId = commentId;
 
+  if (window !== window.top) {
+    const event = {
+      event: 'addComment',
+      params: {
+        padId: pad.getPadId(),
+        comment,
+      },
+    };
+
+    window.top.postMessage(JSON.stringify(event), '*');
```

Comment:
This code sends a message to Prepare, [specifically our Etherpad-wrapper](https://github.com/BridgeU/bridgeu/blob/4a027d44cafc2e24f3621cf4b64133c60e398215/app/assets/javascripts/etherpaddables.js.erb#L45-L72), to allow the comment event to "break out" of the iframe.